dashboard/test/integration/scripts_test.rb

```diff
@@ -16,13 +16,14 @@ class ScriptsTest < ActionDispatch::IntegrationTest
     sign_in @teacher
 
     @unit = create :script, name: 'csp1-2020'
+    create :single_unit_course, unit: @unit, name: 'csp-2020'
```

Comment:
for next time, it's better to omit the name (or any other unique field) and let the factory pick one, unless there's something specific about the name which you are testing. this reduces the chances of unique constraint violations and also makes it clearer that the particular name is not important.

apologies for introducing at least a few violations of this advice into the codebase 😅